Mobile Buy SDK/Mobile Buy SDK/Data/BUYClient.m

```diff
@@ -471,7 +487,13 @@ - (NSURLSessionDataTask *)completeCheckout:(BUYCheckout *)checkout withApplePayT
 	}
 	else {
 		NSString *tokenString = [[NSString alloc] initWithData:token.paymentData encoding:NSUTF8StringEncoding];
-		NSDictionary *paymentJson = @{ @"payment_token" : @{ @"payment_data" : tokenString, @"type" : @"apple_pay" }};
+		NSDictionary *paymentJson = @{
+									  @"payment" : @{
```

Comment:
@davidmuzi there is no model in this case. Are you thinking the checkout? Because I don't think that's within that object's area of responsibility.

In this case, the body is purely a convention of the endpoint, that no code or class outside of this method should care about.